ClientApp/src/Pages/Conservation/Conservation.tsx

```diff
@@ -0,0 +1,94 @@
+import "./Conservation.scss";
+import "bootstrap";
+import React from "react";
+import * as ConservationData from "./ConservationData.json";
+
+export function Conservation(): JSX.Element {
+  return (
+    <div>
+
+      <div className="container-fluid">
+          <h2 className="conservationTitle">Conservation</h2>
+          <p className="text-justify">
+            Some good work has been done in recent decades to reverse the trend of
+            almost inevitable extinction for some species of cetaceans.
+            Unfortunately, whales and dolphins continue to be threatened by:
+          </p>
+      </div>
+
+      <div className="container">
+
+        <div className="row">
+
+          <div className="col-sm">
+            <div className="card">
+              <img src={ConservationData.Pollution.imgSrc} />
+              <div className="card-body">
+                <h5 className="card-title">Pollution</h5>
+                <p className="card-text">{ConservationData.Pollution.text}</p>
+                <a href={ConservationData.Pollution.siteLink} className="btn btn-primary">
+                  Read more
+                </a>
+              </div>
+            </div>
+          </div>
+
+          <div className="col-sm">
+            <div className="card">
+              <img src={ConservationData.Nets.imgSrc} className="card-img-top" alt="..." />
+              <div className="card-body">
+                <h5 className="card-title">Nets</h5>
+                <p className="card-text">{ConservationData.Nets.text}</p>
+                <a href={ConservationData.Nets.siteLink} className="btn btn-primary">
+                  Read more
+                </a>
+              </div>
+            </div>
+          </div>
```

Comment:
So we've got this block of code quite a few times.
I think to avoid the duplication and make it easier to extend we should create a little component that does this for us.

That should make it much easier to edit this card in the future or add new cards etc.

Reply:
We have put this into a map function now